app/views/forever_style_guide/sections/components/navigation/_impersonation_banner.erb

```diff
@@ -0,0 +1,10 @@
+<div class="impersonation_banner color_block-danger">
+  <div class="container-fluid">
+    <div class="color-white pull-left">
+      <span class="hidden-xs">Logged in as <%= current_admin.name %>, but a</span><span class="visible-xs-inline">A</span>cting on behalf of <%= current_user.name %>
+    </div>
+    <%= link_to stop_impersonating_url, class: 'btn btn-sm btn-link btn-cancel_impersonation' do %>
+      <i class="fa fa-times"></i> Stop Managing <%= current_user.name %>
```

Comment:
fa-fw is a shorthand for fixed width, which assigns am em based width to the icon and text-align: center; (http://fontawesome.io/examples/). I added it to the vertical list nav items to ensure consistent display, but don't think it's necessary here.